runtime/src/bank.rs

```diff
@@ -54,6 +54,8 @@ use std::{
 
 pub const SECONDS_PER_YEAR: f64 = (365.25 * 24.0 * 60.0 * 60.0);
 
+const MAX_LEADER_SCHEDULE_STAKES: Epoch = 200; // what value shold this be???
```

Comment:
32 is more than enough, IMHO, it represents 6+months of history

Reply:
I see 3f4064086